fileno, strptime are not __STRICT_ANSI__

--- origsrc/aria2-1.18.9/configure.ac	2015-02-01 07:39:50.000000000 -0600
+++ src/aria2-1.18.9/configure.ac	2015-02-12 12:16:38.451912300 -0600
@@ -129,7 +129,7 @@ AC_LANG([C++])
 PKG_PROG_PKG_CONFIG([0.20])
 
 # Check C++ compiler supports C++0x/C++11 feature
-AX_CXX_COMPILE_STDCXX_11([noext], [mandatory])
+AX_CXX_COMPILE_STDCXX_11([ext], [mandatory])
 
 # i686-w64-mingw32-g++ 4.6 does not support override keyword. For
 # those compilers, define CXX11_OVERRIDE to empty string. Otherwise
